### PR TITLE
Add referral to forum in error messages

### PR DESCRIPTION
**Pull request by lillian542**
Add a message at the end of errors raised in calling the converted circuit, directing users to the forum if they are having trouble.

To merge after #418 (and handling merge conflicts)

### CODE REVIEW

**Comment by lillian542** [comment]
[sc-56367]

File: pennylane_qiskit/converter.py [hunk 1]
```diff
@@ -35,6 +35,8 @@
 
 dagger_map = {"SdgGate": qml.S, "TdgGate": qml.T, "SXdgGate": qml.SX}
 
+referral_to_forum = "\n \nIf you are experiencing difficulties with converting circuits from Qiskit, you can reach out " \
```
Comment (obliviateandsurrender) [inline pennylane_qiskit/converter.py:38]:
```suggestion
referral_to_forum = "\n \nIf you are experiencing any difficulties with converting circuits from Qiskit, you can reach out " \
```
(optional)

**Review by obliviateandsurrender — APPROVED** [review]
Thanks @lillian542! 🚀

File: pennylane_qiskit/converter.py (hunk 1 above)
Comment (lillian542) [reply pennylane_qiskit/converter.py:38]: sounds good!

**Review by timmysilv — APPROVED** [review]

**Pull request merged**